BusinessTripApplication/Service/AreaService.cs

```diff
@@ -29,5 +30,25 @@ public IList<Area> FindAll()
                 throw;
             }
         }
+
+        public Area FindById(int Id)
+        {
+            IList<Area> areas = new List<Area>();
+            try
```

Comment:
why is here a try...catch, with a direct throw in it? i don't understand the purpose of this code